src/Main.java

```diff
@@ -1,5 +1,110 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        System.out.println("Задание 1");
+        byte cat = 5;
+        System.out.println(cat);
+        short dog = 300;
+        System.out.println(dog);
+        int pig = 60000;
+        System.out.println(pig);
+        long starFish = - 1234567891;
+        System.out.println(starFish);
+        float squid = 5.3f;
+        System.out.println(squid);
+        double friedFish = 1.8f;
+        System.out.println(friedFish);
+
+        System.out.println("Задание 2");
+        double a = 27.12f;
+        System.out.println(a);
+        long b = 987678965549L;
+        System.out.println(b);
+        double c = 2.786f;
```

Comment:
```suggestion
        double c = 2.786;
```
почему float записывается в double?